docs/rule/no-whitespace-within-word.md

```diff
@@ -0,0 +1,53 @@
+## no-whitespace-within-word
+
+Using in-line whitespace word formatting results in pages that are in violation of the Web Content Accessibility Guidelines (WCAG 2.0). This approach reflects a choice to ignore [WCAG's officially-documented techniques](https://www.w3.org/TR/WCAG20-TECHS/intro.html) that ensure web accessibility criteria are met, and, unsurprisingly, produces results that are explicitly mentioned in [WCAG's list of common sources of web accessibility failures](https://www.w3.org/TR/WCAG20-TECHS/failures.html).
+
+Specifically, this common whitespace-within-word-induced web accessibility failure is -- from a WCAG perspective -- a failure to successfully achieve [WCAG Success Criterion 1.3.2: Meaningful Sequence](https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-sequence.html). In practice, the predominant issue raised by inline whitespace styling is that the resultant text 'formatting' is entirely visual in nature; the ability to discern the correct manner in which to read the text, and therefore, to correctly comprehend its meaning, is restricted to sighted users.
```

Comment:
> In practice, the predominant issue raised by inline whitespace styling is that the resultant text 'formatting' is entirely visual in nature; the ability to discern the correct manner in which to read the text, and therefore, to correctly comprehend its meaning, is restricted to sighted users

This seems like an **excellent** introduction, and should be _**first**_. Then we can back it up with the spec that agrees with us.